specification/types.md

```diff
@@ -93,3 +93,13 @@ A structure containing the following fields:
 A structure which supports definition of arbitrary properties, with keys of type `string`, and values of type `boolean`, `string`, or `number`.
 
 This structure is populated by a provider for use by an [Application Author](./glossary.md#application-author) (via the [Evaluation API](./glossary.md#evaluation-api)) or an [Application Integrator](./glossary.md#application-integrator) (via [hooks](./sections/04-hooks.md)).
+
+### Provider Events
+
+An enumeration of provider events.
+
+| Event                          | Explanation                                                                                 |
+| ------------------------------ | ------------------------------------------------------------------------------------------- |
+| PROVIDER_READY                 | The provider is ready to perform flag evaluations.                                          |
+| PROVIDER_ERROR                 | The provider signalled an error.                                                            |
+| PROVIDER_CONFIGURATION_CHANGED | A change was made to the backend flag configuration.                                        |
```

Comment:
minor - seems `PROVIDER_SHUTDOWN` is missing

Reply:
Sorry, I meant to remove `PROVIDER_SHUTDOWN` entirely. I'm not sure it's useful enough. I can't imagine why a provider would permanently shutdown and how that would be different than an error.

Reply:
Agree with this :) I noticed the usage and this is the reason to comment